system-tests/test/service_worker_protocol_spec.js

```diff
@@ -56,6 +56,35 @@ describe('capture-protocol', () => {
           expect(parsedProtocolEvents.exceptionThrown).to.be.false
         })
       })
+
+      it(`verifies the types of requests match for a preloaded service worker - ${browser}`, function () {
+        return systemTests.exec(this, {
+          key: 'f858a2bc-b469-4e48-be67-0876339ee7e1',
+          project: 'protocol',
+          configFile: 'cypress-with-service-worker-preloaded.config.ts',
+          // Here we are testing that when a service worker is preloaded it can properly be captured
+          spec: 'protocol.cy.js,service-worker-preloaded.cy.js',
```

Comment:
Do both specs need to run?

Reply:
Updated to only run what is needed for this scenario: [`c4a5906` (#28748)](https://github.com/cypress-io/cypress/pull/28748/commits/c4a5906be5513a75624854f349daf80f93e850a2)